powerauth-push-server/src/main/java/io/getlime/push/controller/rest/AdministrationController.java

```diff
@@ -154,9 +163,12 @@ public AdministrationController(PowerAuthServiceClient powerAuthClient, AppCrede
      * @return Create application response.
      * @throws PushServerException Thrown when request validation fails.
      */
-    @RequestMapping(value = "create", method = RequestMethod.POST)
-    public @ResponseBody ObjectResponse<CreateApplicationResponse> createApplication(@RequestBody ObjectRequest<CreateApplicationRequest> request) throws PushServerException {
+    @PostMapping(value = "create")
+    public ObjectResponse<CreateApplicationResponse> createApplication(@RequestBody ObjectRequest<CreateApplicationRequest> request) throws PushServerException {
         final CreateApplicationRequest requestObject = request.getRequestObject();
+        if (requestObject != null) {
```

Comment:
The logging is not consistent with the one above and this seems a bit weird (if-else just for logging). We should probably improve the code by checking for `null` only (duplicate validation done later in validator code), throwing an exception in case `null` occurs, and then do the regular logging behind it, like so:

```java
if (requestObject == null) {
    throw new PushServerException("Request object must not be epmty");
}
logger.info("Received createApplication request, application ID: {}", requestObject.getAppId());
```